### ISSUE TITLE
Regarding the initialization and image preprocessing

### ISSUE THREAD

**Issue by laukun** [DrSleep/tensorflow-deeplab-lfov#21]
1. Did you reproduce a mIoU of 57.% with all the parameters you provided in the train.py? I just got 52.%....
2. if there any difference between initializing the deeplab-lfov model with vgg16.ckpt weights pretrained on ImageNet (for the conv1-conv5, nomal distribution for the rest)  and with the .caffemodel provided by author?
3. why images are not loaded to a range of [0,1] and normalized with a standard mean and std if model is initialized with vgg pretrained weights? will it be a cause to the resulted mIoU about 5% lower than author's?

Thank you!
